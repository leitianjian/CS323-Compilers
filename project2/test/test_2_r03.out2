Error type 3 at Line 6: ttt variable is redefined in the same scope
