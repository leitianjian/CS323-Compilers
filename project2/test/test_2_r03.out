Error type 3 at line 6: ttt variable is redefined in the same scope
